### PR TITLE
Aha/ox minifx

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/resources/component-biz.conf [hunk 1]
```diff
@@ -654,9 +655,10 @@ security {
             permission-system-load = true
             permission-tasks = true
             permission-manage-all-processes = true
-            feature-provide-jobs = true
```
Comment (meg1502) [inline src/main/resources/component-biz.conf]: Not Subject of this commit ?

File: src/main/resources/component-biz.conf [hunk 2]
```diff
@@ -654,9 +655,10 @@ security {
...
             permission-view-rate-limits = true
             permission-delete-user-accounts = true
+
```
Comment (meg1502) [inline src/main/resources/component-biz.conf]: is this extra line intended?

File: src/main/resources/component-biz.conf (hunk 1 above)
Comment (andyHa) [reply src/main/resources/component-biz.conf]: yep, this was a left-over of a pervious refactoring - the feature is long gone...

File: src/main/resources/component-biz.conf (hunk 2 above)
Comment (andyHa) [reply src/main/resources/component-biz.conf]: yep for readability

**Pull request merged**